android/build.gradle

```diff
@@ -112,6 +112,8 @@ android {
 
   packagingOptions {
     excludes = ["**/libc++_shared.so"]
+    exclude "META-INF/**"
+    pickFirst '**/*.so'
```

Comment:
this line is related to the same problem that  i found here, https://github.com/mrousavy/react-native-vision-camera/issues/365

but don't know if this is the better fix for that,